docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+Since the ASB is a long running task we specify the person running an ASB as service provider.
+
+### ASB discovery
+
+Currently, there is no automated discovery for service providers running an ASB.
+A service provider has to manually provide the connection details to users that will run the CLI.
+
+Currently the connection details consist of:
+
+- Multiaddress
+- Peer-ID
+
+The Peer-ID is printed upon startup of the ASB.
+The multi-address is the IP-address or URL (if DNS entry configured) of the service provider.
+
+Each service provider running an ASB can decide how/where to share these connection details
+
+![Service Provider scenarios](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/xmr-btc-swap/04a65005428c34891451762a94035df5754e6c19/docs/asb/diagrams/cli-asb-overview.puml)
+
+Eventually, more elaborate discovery mechanims can be added.
+
+### Setup Details
+
+In order to understand the different components of the ASB and CLI better here is a component diagram showcasing the ASB and CLI setup using public Bitcoin and Monero infrastructure:
+
+![Service Provider scenarios](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/xmr-btc-swap/7afa953efd398d65d54b4488ec11d40a8647ab24/docs/asb/diagrams/cli-asb-components-asb-pub-nodes.puml)
+
+Contrary, here is a diagram that showcases an ASB running it's own blockchain infrastructure:
+
+![Service Provider scenarios](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/xmr-btc-swap/7afa953efd398d65d54b4488ec11d40a8647ab24/docs/asb/diagrams/cli-asb-components-asb-self-hosted.puml)
+
+The diagram shows that the `asb` group (representing the `asb` binary) consists of three components:
+
+1. Monero Wallet
+2. Bitcoin Wallet
+3. ASB
+
+The `ASB` depicted in the diagram actually consists of multiple components (protocol impl, network communication, ...) that sums up the functionality to execute concurrent swaps in the role of Alice.
+
+#### Monero Wallet Setup
+
+The ASB uses the running Monero wallet RPC to create / open Monero wallets.
+Currently you cannot connect to an existing Monero wallet, but the ASB will create the wallet `asb-wallet` upon intial startup.
+In order to accept trades with a CLI you will have to send XMR to that wallet.
+The wallet's address is printed upon startup of the ASB.
+Currently the `asb-wallet` does not have a password.
+
+Upon startup of the ASB the `asb-wallet` is opened in the wallet RPC.
+You can then interact with the wallet RPC for basic wallet management as well.
+
+#### Bitcoin Wallet Setup
+
+The ASB has an internall managed Bitcoin wallet.
+The Bitcoin wallet in created upon initial startup and stored in the data folder of the ASB (configured through initial startup wizard).
+
+Interactions with the Bitcoin wallet are still work in progress:
```

Comment:
I would just write "we are working on adding withdraw and balance to the bitcoin balance {GitHub link to PR}"